Update index.php

salsabillahfebridha opened the pull request:
Backend loginnya jadi satu disini, file loginMhs.php sama login Adm.php diapus

salsabillahfebridha committed:
Update index.php

Backend loginnya jadi satu disini, file loginMhs.php sama login Adm.php diapus

dianingtass reviewed (changes requested):
Login bikin bisa pake nim sama email (jangan email doang) biar nim langsung bisa diambil jg sama page profile.

salsabillahfebridha committed:
Revisi index.php

Login bisa make nim/email (mahasiswa) dan nip/email (admin). Ngirim data ke page profilenya make session

dianingtass reviewed (approved):
Sip tqtq

dianingtass committed:
Merge branch 'main' into page/index
